src/expr/src/vector_op/cast.rs

```diff
@@ -68,72 +67,6 @@ pub fn str_to_timestamp(elem: &str) -> Result<Timestamp> {
     ))
 }
 
-#[function("cast(varchar) -> bytea")]
```

Comment:
This is needed for the following case, where it cannot be evaluated in frontend:
```
create table t (s varchar);
insert into t values ('a');
select s::bytea from t;
```

The solution is to forward it to the actuall implementation:
```
#[function("cast(varchar) -> bytea")]
pub fn str_to_bytea(elem: &str) -> Result<Box<[u8]>> {
  risingwave_common::cast::str_to_bytea(elem)
}
```

Reply:
yep, I just found the unit tests don't work and add it.